hack/clean-up-projects.sh

```diff
@@ -1,9 +1,3 @@
 #!/bin/bash
 
-if [[ -n $1 ]]; then
-    echo "param provided to script, deleting pr projects"
-    oc delete project labs-ci-cd-pr-$1 labs-dev-pr-$1 labs-demo-pr-$1
-else
-    echo "no param provided to script, deleting default projects"
-    oc delete project labs-ci-cd labs-dev labs-demo
-fi
+oc get projects|grep labs|awk '{print $1}'|xargs oc delete projects
```

Comment:
I like where you are going, but in a cluster that has a lot of `labs` projects, this doesn't work. below is the output of the commands without the delete. I only want to delete the `pr` projects to clean up after CI ran. I want the others to stay.
```
[jholmes@fedora27 Code]$ oc get projects|grep labs|awk '{print $1}'
labs-big-data
labs-ci-cd
labs-ci-cd-pr-130
labs-data-analytics
labs-demo
labs-demo-pr-130
labs-dev
labs-dev-pr-130
labs-devryan-test
```